### ISSUE TITLE
hyperparameter optimalisation

### ISSUE THREAD

**Issue by joostgrunwald** [joostgrunwald/MetRoBert#15]
- learning rate
- hidden layers
- random seed
- warming up batches
- total batches
